contracts_use_cases.html

```diff
@@ -660,14 +668,15 @@ <h2>Use cases &mdash; table</h2>
 
 <tr><td></td>
   <td>C++ Library Developer</td>
-  <td>Reduce need for client recompilation</td>
-  <td>Insulate contract conditions with the function body</td>
+  <td>Control the tradeoff between need for client recompilation and contract condition visibility</td>
+  <td>Insulate contract conditions with the function definition</td>
   <td></td></tr>
 
 <tr><td></td>
   <td>C++ Library Developer</td>
-  <td>Reduce need for client recompilation</td>
-  <td>Support traditional separation of definition and declaration</td>
+  <td>Control the tradeoff between need for client recompilation and contract condition visibility</td>
+  <td>Insulate the function body but not the contract conditions (e.g., so static analysis tools can view the contract conditions without
+    needing to process the function body) but still outside the class definition to avoid cluttering my public interface.
```

Comment:
Because noise.  I'm going to trim this down.   Part of this is to motivate a need to redeclare member functions so that contract conditions on member functions can be put outside of the class definition but still "at the bottom of the header" where they won't clutter the human-readable part of the header.